app/src/main/java/org/groundplatform/android/model/geometry/Geometry.kt

```diff
@@ -92,13 +93,27 @@ data class LineString(val coordinates: List<Coordinates>) : Geometry {
   override val area: Double
     get() = 0.0
 
+  val tooltipDistance: Double?
```

Comment:
Does "tooltip distance" make sense as a property of all geometries? This doesn't seem to be something specific to geometries at all and should be calculated in domain layer.